defines.php

```diff
@@ -140,7 +140,23 @@
  * Define for vipgoci_git_diffs_fetch()
  */
 
-define( 'VIPGOCI_GIT_DIFF_CALC_CHANGES',		array ('+' => 'additions', '-' => 'deletions') ); 
+define( 'VIPGOCI_GIT_DIFF_CALC_CHANGES',		array ('+' => 'additions', '-' => 'deletions') );
 define( 'VIPGOCI_GIT_DIFF_DATA_SOURCE_GIT_REPO',	'local-git-repo' );
 define( 'VIPGOCI_GIT_DIFF_DATA_SOURCE_GITHUB_API',	'github-api' );
 
+
+
+/**
+ * Define file number of lines limit
+ */
+define( 'VIPGOCI_SKIPPED_FILES',	        'skipped-files'	);
+
+define( 'VIPGOCI_VALIDATION_MAXIMUM_LINES_LIMIT',   15000   );
```

Comment:
Maybe use this as a default, but then be able to customise the number on the command-line or via environmental options? I do see us possibly changing this number, would be good not having to change the code to do that.